server/src/lib/validation/random-clips-count.ts

```diff
@@ -0,0 +1,13 @@
+import { AllowedSchema } from 'express-json-validator-middleware';
+
+export const randomClipsCountSchema: AllowedSchema = {
```

Comment:
Since the schema can actually be used across different routes, it might be better to use a more generic name for it, like `clipSchema`.